redis/asyncio/cluster.py

```diff
@@ -318,8 +337,7 @@ def __init__(
         self.read_from_replicas = read_from_replicas
         self.reinitialize_steps = reinitialize_steps
         self.cluster_error_retry_attempts = cluster_error_retry_attempts
-        self.connection_error_retry_attempts = connection_error_retry_attempts
-
+        self.retry = retry
```

Comment:
Please update `__slots__` as well.

Reply:
Reverting these deletion to maintain backward compatibility

Reply:
I don't think this line is required. It will overwrite the default implementation on line 312.

Reply:
Not resolved.